### PR TITLE
Automatically Install WordPress and add WooCommerce along with recommended plugins/themes using the Deploy Product Quicksilver Hook

### PR DESCRIPTION
**Pull request by populist**
This platform hook will use wp-cli to automatically install WP using admin/admin as the credentials and using the site name + user email address from Pantheon's settings.

### CODE REVIEW

File: private/scripts/install_wp.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+$admin_email = $_POST['user_email'];
+$site_url = 'https://' . $_SERVER['HTTP_HOST'];
+$title = $_ENV['PANTHEON_SITE_NAME'];
+$admin_user = 'admin';
+$admin_passwd = 'admin';
```
Comment (ataylorme) [inline private/scripts/install_wp.php]: We should generate a random password.

File: private/scripts/install_wp.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+$admin_email = $_POST['user_email'];
+$site_url = 'https://' . $_SERVER['HTTP_HOST'];
+$title = $_ENV['PANTHEON_SITE_NAME'];
+$admin_user = 'admin';
```
Comment (ataylorme) [inline private/scripts/install_wp.php]: Lets use user email rather than the generic `admin`

File: private/scripts/install_wp.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+$admin_email = $_POST['user_email'];
+$site_url = 'https://' . $_SERVER['HTTP_HOST'];
```
Comment (ataylorme) [inline private/scripts/install_wp.php]: Can we use `PANTHEON_ENVIRONMENT` and `PANTHEON_SITE_NAME` to construct the URL?

Comment (populist) [reply private/scripts/install_wp.php]: $site_url now is defined with PANTHEON_ENVIRONMENT and PANTHEON_SITE_NAME

File: private/scripts/install_wp.php (hunk 2 above)
Comment (populist) [reply private/scripts/install_wp.php]: Admin Username is Now User Email

File: private/scripts/install_wp.php (hunk 1 above)
Comment (populist) [reply private/scripts/install_wp.php]: Admin Password is now Randomly Generated Thanks to WP CLI

**Comment by populist** [comment]
Merging in the Auto Install. Let's hope for magic!

**Pull request closed without merging**